runtime/rococo/src/lib.rs

```diff
@@ -559,6 +559,7 @@ impl xcm_executor::Config for XcmConfig {
 	type Call = Call;
 	type XcmSender = xcm_sender::RelayChainXcmSender<Runtime>;
 	type AssetTransactor = LocalAssetTransactor;
+	type HrmpExecutor = Hrmp;
```

Comment:
this only makes sense for the relay chain to have because it is managing the channels, maybe need to have an () impl for parachains